### PR TITLE
Improve editing experience.

### PR DESCRIPTION
**Pull request by MrCoder**

### CODE REVIEW

File: src/components/DiagramFrame/SeqDiagram/MessageLayer/MessageLabel.vue [hunk 1]
```diff
@@ -3,8 +3,8 @@
     title="Double click to edit"
     class="px-1 cursor-text right hover:text-skin-message-hover hover:bg-skin-message-hover"
     :class="{
-      'py-1 px-2 ml-1 cursor-text': editing,
```
Comment (MrCoder) [inline src/components/DiagramFrame/SeqDiagram/MessageLayer/MessageLabel.vue:6]: It adds unwanted padding that makes the editing box jump.
image

File: src/components/DiagramFrame/SeqDiagram/MessageLayer/MessageLabel.vue [hunk 2]
```diff
@@ -3,8 +3,8 @@
...
-      'absolute right-1/2 translate-x-1/2 bottom-0': editing && !isSelfAsync,
```
Comment (MrCoder) [inline src/components/DiagramFrame/SeqDiagram/MessageLayer/MessageLabel.vue:7]: I don't know why we needed this either.

Comment (dontry) [reply src/components/DiagramFrame/SeqDiagram/MessageLayer/MessageLabel.vue:7]: This makes sure the text will always stay center when editing.

File: src/components/DiagramFrame/SeqDiagram/MessageLayer/MessageLabel.vue (hunk 1 above)
Comment (dontry) [reply src/components/DiagramFrame/SeqDiagram/MessageLayer/MessageLabel.vue:6]: I aimed to create more space for the editor box. Should this adjustment cause any instability, feel free to eliminate the styles entirely.

File: src/components/DiagramFrame/SeqDiagram/MessageLayer/Block/Statement/Message/Message.vue [hunk 3]
```diff
@@ -13,7 +13,7 @@
     ref="messageRef"
   >
     <div
-      class="name group flex-grow relative text-sm hover:whitespace-normal hover:text-skin-message-hover hover:bg-skin-message-hover"
```
[the remark this answers is not in the document]
Comment (dontry) [reply src/components/DiagramFrame/SeqDiagram/MessageLayer/Block/Statement/Message/Message.vue:16]: Yes, you can remove this one.

File: src/components/DiagramFrame/SeqDiagram/MessageLayer/MessageLabel.vue (hunk 2 above)
Comment (MrCoder) [reply src/components/DiagramFrame/SeqDiagram/MessageLayer/MessageLabel.vue:7]: Without those classes, the message text always stays at its original position and when we commit the changes the message is also staying where it is. I feel that is a better experience.

Comment (dontry) [reply src/components/DiagramFrame/SeqDiagram/MessageLayer/MessageLabel.vue:7]: I want to mention that using `position: absolute;` is essential for editing text. Without it, the arrow may get displaced by long text. With position set to absolute, centering the editing text becomes much simpler compared to maintaining its original position.

**Pull request closed without merging**